ls1intum/Artemis
# `Assessment`: Show legend warning element only for tutors when assessing example submissions for modeling exercises

### Checklist
#### General


#### Server
#### Client
#### Changes affecting Programming Exercises

[ ] * - not needed

### Motivation and Context


Legend view was introduced for example submissions modeling editor. Right now for both instructors and tutors that are reading or assessing the example submissions the "Incorrect assessment" legend element is shown.
This PR makes it to be shown only for tutors that are assessing the example submission as it only makes sense to show during the training process.

### Description

The legend element is shown only for `toComplete` being set to true (which means that the example submission needs to be assessed)

### Steps for Testing

Prerequisites:
- 1 Instructor
- 1 Tutor

1. Log in to Artemis as an Instructor
2. Navigate to Course Administration
3. Navigate to Modeling Exercise or Create your own
4. Create an Example Submission and example assessment and don't check Used For Tutorial. 
5. Create another Example Submission and example assessment and check Used For Tutorial. 
6. The legend element "Incorrect assessment" **should not be shown** when creating the example assessment.
---

1. Log in to Artemis as Tutor
2. Navigate to Course Administration
3. Navigate to Modeling Exercise Assessment Dashboard
4. Read an Example Submission -> the legend element **should not be shown**
5. Assess an Example Submission -> the legend element **should be shown**

### Review Progress





#### Code Review
#### Manual Tests

### Screenshots


#### Read
<img width="1863" alt="Screen Shot 2021-11-22 at 00 13 56" src="https://user-images.githubusercontent.com/51077603/142783350-828db4ea-0905-4c38-967d-e39cd78ec629.png">

#### Assess
<img width="1871" alt="Screen Shot 2021-11-22 at 00 14 16" src="https://user-images.githubusercontent.com/51077603/142783364-edb466a9-668f-4768-a6a5-06c33cc80dfe.png">
